home/tests/test_insert_question_feature.py

```diff
@@ -465,6 +475,19 @@ def test_post_question_without_title_with_client(
              Each tag must have between 2-20 characters. And cannot contain special characters (like * # " etc.) """,
                 ),
                 # User entered all the required fields and invalid tags (more than 5 tags)
+                (
+                    (
+                        {
+                            "title": "Question in Math",
+                            "subject": 1,
+                            "sub_subject": 10,
+                            "content": "How much is it 1+1?",
+                            "grade": Grade.GRADE7,
+                        }
+                    ),
+                    """The sub-subject doesn't match the subject""",
```

Comment:
Is there a test of matching sub-subject to a certain subject?

Reply:
there isn't, I'll add